CONTRIBUTING.md

```diff
@@ -8,17 +8,21 @@ This project is made up of a cross section of different parts of Mozilla, includ
 
 | - | Name | Github Handle | Position |
 | - | ---- | ------------- | -------- |
-| ![][clarkbw] | Bryan Clark | [@clarkbw](https://github.com/clarkbw) | Firefox DevTools Product Manager |
+| ![][digitarald] | Harald Kirschner | [@digitarald](https://github.com/digitarald) | Firefox DevTools Product Manager |
 | <img src="https://avatars.githubusercontent.com/mstange?size=56" width="56" height="56" /> | Markus Stange | [@mstange](https://github.com/mstange) | Firefox Platform Engineer |
 | ![][gregtatum] | Greg Tatum | [@gregtatum](https://github.com/gregtatum) | Firefox DevTools Engineer |
 | ![][julienw] | Julien Wajsberg | [@julienw](https://github.com/julienw) | Firefox DevTools Engineer |
+| ![][brisad] | Michael Hoffmann | [@brisad](https://github.com/brisad) | Firefox DevTools Engineer |
```

Comment:
Michael is a part of the team, but not a Firefox DevTools Engineer. 
He is an Engineer and Contributor.

Reply:
@zoepage so just "Engineer and Contributor" should work, right?